<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QDialog" name="Editor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Write a Journal Entry</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icons/robojournal-icon.png</normaloff>:/icons/robojournal-icon.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="WordCount">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>446</y>
     <width>119</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Character Count: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="DocumentMode">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>446</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="CharacterCount">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>446</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>wordcount</string>
   </property>
  </widget>
  <widget class="QLabel" name="RemainingCharacters">
   <property name="geometry">
    <rect>
     <x>484</x>
     <y>446</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>remaining</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>567</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="GUI" stretch="0,1">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="Controls" rowstretch="1,0" columnstretch="0,1,1,1">
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Entry Title</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="EntryTitle">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="maxLength">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Entry Date</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="EntryDate">
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MonthSection</enum>
        </property>
        <property name="displayFormat">
         <string>MMMM dd, yyyy</string>
        </property>
        <property name="timeSpec">
         <enum>Qt::LocalTime</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="Cancel">
        <property name="text">
         <string>Abort</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="PostEntry">
        <property name="text">
         <string>Post New Entry</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons/pencil.png</normaloff>:/icons/pencil.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="DocumentArea">
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="verticalSpacing">
       <number>7</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="CutButton">
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Cut</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="CopyButton">
        <property name="font">
         <font>
          <family>Arial</family>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Copy</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons/copy.gif</normaloff>:/icons/copy.gif</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="PasteButton">
        <property name="font">
         <font>
          <family>Arial</family>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Paste</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons/paste_plain.png</normaloff>:/icons/paste_plain.png</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="UndoButton">
        <property name="text">
         <string>Undo</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons/arrow_undo.png</normaloff>:/icons/arrow_undo.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="RedoButton">
        <property name="text">
         <string>Redo</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons/arrow_redo.png</normaloff>:/icons/arrow_redo.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="FormatButton">
        <property name="text">
         <string>Style</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons/text_dropcaps.png</normaloff>:/icons/text_dropcaps.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QTextEdit" name="EntryPost">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="documentTitle">
         <string/>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
